protocol/x/clob/abci.go

```diff
@@ -20,6 +21,11 @@ func BeginBlocker(
 	ctx sdk.Context,
 	keeper types.ClobKeeper,
 ) {
+	ctx = log.AddPersistentTagsToLogger(ctx,
+		log.Handler, log.BeginBlocker,
+		log.BlockHeight, ctx.BlockHeight(),
+	)
+
```

Comment:
nit: each arg on it's own line

Reply:
i think it's a bit more clear here since it's `key, val` alternating here